data/convert_to_triples.R

```diff
@@ -0,0 +1,442 @@
+library(dplyr)
+library(tidyr)
+library(readr)
+library(stringr)
+
+reformatted_references <- read_csv("data/APD_references.csv") %>%
+  mutate(across(where(is.character), \(x) stringr::str_replace_all(x, "\"", "'"))) %>%
+  mutate(
+    Entity = paste0("<", Entity, ">"),
+    label = paste0("\"", label, "\"", "^^<xsd:string>"),
```

Comment:
the xsd:string type is the default , so this can be omitted on all strings.  

More importantly though, note that the prefix `xsd:` has not yet been defined, and prefixes are not supported when we generate the nquads format, so all type declarations must use a full URI `<http://www.w3.org/2001/XMLSchema#string>`